about.md

```diff
@@ -7,25 +7,32 @@ image: about-background
 
 ## Who are we?
 
-Iron Claw is a [First Robotics Competition](http://www.usfirst.org/roboticsprograms/frc) (FRC)
-team run by the Los Gatos High School Robotics Club. Our team number is 972 and our rookie year is 2016!
-We hope to see all of our fellow teams at Silicon Valley Regional in April!
+Iron Claw is a <a href="http://www.usfirst.org/roboticsprograms/frc">FRC</a>
+team run by the Los Gatos High School Robotics Club. This year (2016), we
+will be restarting team 972 at LGHS. We hope to see all of our fellow teams
+and students at Silicon Valley Regional in April 2016!
 
 ## Our History
 
-Team 972 was founded as "Steel Paw" in 2002. You can see a video of their rookie year robot at Silicon Valley
+Team 972 was founded as "Steel Paw" in 2002. You can see a video of their rookie year robot at the Silicon Valley
 Regional [here](https://www.youtube.com/watch?v=uSXVit1UKro"). 972's name later changed to "Iron Paw".
-Unfortunately, team 972 stopped competing in 2008 due to a lack of interest and the retirement of the head mentor.
+Unfortunately, team 972 stopped competing in 2008 due to a lack of interest and the retirement of the lead mentor.
 Students traveled to Pioneer High School's team 668, the [Apes of Wrath](http://www.apesofwrath668.org")
 to continue participating in robotics.
 
-Soon, enough interest built up in LGHS for the Robotics Club to form. We participated as team 6730 in the
-FIRST Tech Challenge (FTC) in 2012. FTC is a smaller-scale and lower-cost robotics competition. In 2014, the
-program grew enough for LGHS to field a second FTC team, 9097. We still participated in FRC with Pioneer
-High School.
+Soon, enough interest built up for the Robotics Club to form at LGHS. We participated as team 6730 in the
+FIRST Tech Challenge (FTC) in 2012. FTC is a smaller-scale and lower-cost robotics competition. In fall of
+2014, the program grew enough for LGHS to field a second FTC team, 9097. At the end of the 2015 FRC season,
+the Los Gatos High School Robotics Club decided to split off from Pioneer High School and the Apes of Wrath
+due to many students' inability to transport on a daily basis and our team's growing size.
 
-We are now restarting team 972 as "Iron Claw". Our mission is to grow interest in STEM in the Los Gatos community
+We are now restarting team 972 as "Iron Claw", featuring a fresh, new team while still retaining characteristics
```

Comment:
`featuring a fresh, new team while still retaining characteristics of our old team` sounds odd to me. Maybe remove or rephrase this